src/Sylius/Behat/Context/Ui/Admin/ManagingAdministratorsContext.php

```diff
@@ -333,6 +342,17 @@ public function iShouldSeeTheAvatarImageInTheTopBarNextToMyName(string $avatar):
         Assert::true($this->topBarElement->hasAvatarInMainBar($avatar));
     }
 
+    /**
+     * @Given /^I should not see the "([^"]*)" avatar image in the top bar next to my name$/
```

Comment:
Could be written with turnip `@Then I should not see the :avatar avatar image in the top bar next to my name`